tests/qir_test.py

```diff
@@ -542,6 +543,18 @@ def test_qir_from_pytket_circuit_and_pyqir_gateset(
         assert call_ry in data
         assert call_rz in data
 
+    def test_classical_arithmetic(
```

Comment:
I don't understand what this is testing. Aren't we just reading the same file twice?

Reply:
No. There is a test file in `/qir_test_files` that gets read and a similarly named one in the current directory that gets generated in the [fixture](https://github.com/CQCL/pytket-qir/blob/4b71cbfdbf434b4a6cac6f4aada274d75bdca0a2/tests/conftest.py#L228), read in the test and deleted in the fixture.